desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -0,0 +1,418 @@
+
+        return trade != null ? trade.getOffer() : null;
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Life cycle
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void initialize() {
+        Label label = new AutoTooltipLabel(Res.get("support.filter"));
+        HBox.setMargin(label, new Insets(5, 0, 0, 0));
+        HBox.setHgrow(label, Priority.NEVER);
+
+        filterTextField = new InputTextField();
+        Tooltip tooltip = new Tooltip();
+        tooltip.setShowDelay(Duration.millis(100));
+        tooltip.setShowDuration(Duration.seconds(10));
+        filterTextField.setTooltip(tooltip);
+        HBox.setHgrow(filterTextField, Priority.NEVER);
+
+        filterTextField.setText("open");
+
+        setupTable();
+    }
+    @Override
+    protected void activate() {
+        super.activate();
+
+        observableList = FXCollections.observableArrayList();
+        observableList.addAll(openOfferManager.getSignedOffers());
+
+        SortedList<SignedOffer> sortedList = new SortedList<>(observableList);
+        sortedList.comparatorProperty().bind(tableView.comparatorProperty());
+        tableView.setItems(sortedList);
+
+        contextMenu = new ContextMenu();
+        MenuItem item1 = new MenuItem(Res.get("support.contextmenu.penalize.msg",
+                Res.get("shared.maker")));
+        contextMenu.getItems().addAll(item1);
+
+        tableView.setRowFactory(tv -> {
+            TableRow<SignedOffer> row = new TableRow<>();
+            row.setOnContextMenuRequested(event -> {
+                contextMenu.show(row, event.getScreenX(), event.getScreenY());
+            });
+            return row;
+        });
+
+        item1.setOnAction(event -> {
+            selectedSignedOffer = tableView.getSelectionModel().getSelectedItem();
+            Offer offer = fetchOffer(selectedSignedOffer);
+            log.debug("Found {} matching signed offer.", (offer != null ? offer.getId() : null));
+            if(offer != null) {
+                new Popup().warning(Res.get("support.prompt.signedOffer.penalty.msg",
+                                //Hack to remove BTC uses String.replace()
+                                HavenoUtils.formatToXmr(offer.getMakerFee()),
+                                HavenoUtils.formatToXmr(offer.getAmount().subtract(offer.getMakerFee())),
+                                HavenoUtils.formatToXmr(offer.getTxFee()),
+                                selectedSignedOffer.getReserveTxHash(),
+                                selectedSignedOffer.getReserveTxKeyImages())
+                        ).onAction(() -> OfferViewUtil.submitTransactionHex(xmrWalletService, tableView,
+                                selectedSignedOffer.getReserveTxHex())).show();
+            } else {
+                new Popup().error(Res.get("support.prompt.signedOffer.error.msg")).show();
```

Comment:
Now getting this error when trying to sign an offer.

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/13068859/216342490-f85cb870-e185-4233-96f6-0c2241620316.png">

Signed offers do not have associated trades yet, so this error does not apply here.